src/loaders/ObjectLoader.js

```diff
@@ -741,6 +753,22 @@ ObjectLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 						object.background = new Color( data.background );
 
+					} else {
+
+						object.background = getTexture( data.background );
+
+					}
+
+				}
+
+				if ( data.environment !== undefined ) {
+
+					var texture = getTexture( data.environment );
+
+					if ( texture instanceof CubeTexture ) {
```

Comment:
I would leave this check out for now since it's not yet clear how HDR textures/env maps should be serialized. These might now be instances of `CubeTexture`.